spec/feature/books/index_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe 'books#index', type: :feature do
```

Comment:
Please use feature specs aliases: `describe, context` => `feature`, `it` => `scenario`